src/libraries/System.Security.Cryptography/src/System/Security/Cryptography/KmacXof256.cs

```diff
@@ -19,8 +19,7 @@ namespace System.Security.Cryptography
     /// </remarks>
     public sealed class KmacXof256 : IDisposable
     {
-        // Some platforms have a mutable struct for LiteKmac, do not mark this field as readonly.
-        private LiteKmac _kmacProvider;
+        private readonly LiteKmac _kmacProvider;
```

Comment:
LiteKmac is a `readonly struct` on all platforms so the mutability comment was inaccurate (it was accurate earlier on). I fixed it for the non-XOF KMACs, but just now fixed it for the XOF ones.